Cargo.toml

```diff
@@ -51,3 +52,7 @@ create-release = false
 [profile.dist]
 inherits = "release"
 lto = "thin"
+
+[profile.release.package.sql-experimental]
+debug = false
+strip = true
```

Comment:
@LucioFranco just wondering, does Rust build chain support splitting debuginfo to a separate file? So that we get the small binary, but can still unstrip (`e.g. with eu-unstrip`) for debugging sessions